### DISCUSSION TITLE
Spotify renderer not visible on android

### DISCUSSION THREAD

**Discussion by Sigmun** [rern/rAudio#1056] [Fixed Issues] [answered]
I've followed all the steps on configuring spotify renderer but I cannot make it working on my android phone. It works from my computer only if I use macos app, not webapp.

Looking on server side, I have le following service status :
```
spotifyd 0.3.3-1
# cat /etc/spotifyd.conf
[global]
bitrate = 320
onevent = "/srv/http/bash/spotifyd.sh"
use_mpris = false
backend = "alsa"
device = "default:CARD=sndrpijustboomd"
mixer = "Digital"
control = "hw:0"
volume_controller = "alsa"

spotifyd.service- A spotify playing daemon
     Loaded: loaded (/etc/systemd/system/spotifyd.service; enabled; preset: disabled)
     Active: active (running) since Sun 2022-10-09 00:08:48 UTC; 3min 52s ago
   Main PID: 785 (spotifyd)
      Tasks: 2 (limit: 1062)
        CPU: 104ms
     CGroup: /system.slice/spotifyd.service
             └─785 /usr/bin/spotifyd --no-daemon

Oct 09 00:08:48 rAudio spotifyd[785]: Using alsa volume controller.
Oct 09 00:08:48 rAudio spotifyd[785]: Failed to register IPv6 receiver: Os { code: 97, kind: Uncategorized, message: "Address family not supported by protocol" }
Oct 09 00:08:48 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:08:49 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:08:52 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:09:01 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:09:28 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:10:50 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:12:38 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
Oct 09 00:12:39 rAudio spotifyd[785]: couldn't parse packet from 192.168.1.146:5353: query type 65 is invalid
```

**Comment by Sigmun** [comment]
I also wonder why we have to follow this complex procedure to have spotify connect when volumio offers a spotify plugin that is simple and allows multiple devices connection.

**Reply by rern** [reply]
It used to be simple, with single shared codes, but the [Rate Limits](https://developer.spotify.com/documentation/web-api/guides/rate-limits/) is waiting around the corner.
With the (not that) complex procedure to let each user use their own codes, the corner is gone.

**Reply by Sigmun** [reply]
OK but this does totally answer my question. Why couldn't it be as simple as it is on volumio?

BTW, you the issue was my mistake. I use TrackerControl app as local VPN to block undesired traffic. In that case, Spotify was blocked from finding spotify connect server.

**Comment by Sigmun** [comment, marked as answer]
the issue was my mistake. I use TrackerControl app as local VPN to block undesired traffic. In that case, Spotify was blocked from finding spotify connect server. You can close this discussion
